addons/account/tests/test_chart_template.py

```diff
@@ -318,6 +318,44 @@ def local_get_data(self, template_code):
             {'name': 'Tax 4'},
         ])
 
+    def test_update_taxes_removal(self):
```

Comment:
I would have expected the test to either:
* unlink a tax before calling `try_loading` again and make sure that the unlinked tax isn't recreated
* add a new tax to the template data and make sure that the new one is not created calling `try_loading`

What is done below doesn't make sense: "Tax 2" existed before calling `try_loading` and it shouldn't be removed when calling it.

Reply:
Yes actually I have modified the hole test as there was a problem with it  :)